doc/classes/VisualServer.xml

```diff
@@ -909,6 +923,7 @@
 			<argument index="2" name="max_z" type="int">
 			</argument>
 			<description>
+				Sets the z range of objects that will be affected by this light. Equivalent to [member Light2D.range_z_min] and [member Light2D.range_z_max].
```

Comment:
We currently tend to use capital Z when referring to the Z component in the docs. It's not fully consistent though, might need harmonization:
```
$ rg "[ ]Z[ ,]" | wc -l
53
$ rg "[ ]z[ ,]" | wc -l
5
```